LibreNMS/Alert/Transport/Ciscospark.php

```diff
@@ -19,21 +19,29 @@
 class Ciscospark extends Transport
 {
     protected string $name = 'Cisco Webex Teams';
+    // This is the total length minus 4 bytes for ellipses.
+    private static int $MAX_MSG_SIZE = 7435;
 
     public function deliverAlert(array $alert_data): bool
     {
         $room_id = $this->config['room-id'];
         $token = $this->config['api-token'];
-        $url = 'https://api.ciscospark.com/v1/messages';
+        $url = 'https://webexapis.com/v1/messages';
         $data = [
             'roomId' => $room_id,
         ];
 
+        if (strlen($alert_data['msg']) > Ciscospark::$MAX_MSG_SIZE) {
```

Comment:
Hi @xorrkaz 
You should probably move the truncate code after the block that removes blank lines, and tags.

Reply:
[Face palm].  Yup.  Good call!